report_pre_thesis/src/sections/related_work.tex

```diff
@@ -1,14 +1,43 @@
+
+The research on the effect of training set size offers relevant considerations for our work. Their analysis of different error metrics for varying dataset sizes provides a good framework when analyzing the balance between data quantity and model structure in our efforts to minimize overall error
+
+\citeauthor{castorena_deep_2021} developed a deep spectral convolutional neural network (CNN) for processing and analyzing laser-induced breakdown spectroscopy (LIBS) signals.
+Their CNN is designed to first remove uncertainties from sensor data and then assess the chemical content qualitatively and quantitatively from a sample's LIBS signal.
+Their experiments showed that their method surpasses existing ones used by the Mars Science Laboratory for pre-processing and calibration.
+Notably, their CNN operates in real-time and does not require additional information like dark current, system response, temperature, or range.
+For chemical content estimation, their CNN matches current techniques' performance and even exceeds them when utilizing an end-to-end learning approach, all the while reducing prior information requirements and being ready for immediate deployment\cite{castorena_deep_2021}.
+
+Their development of a deep spectral CNN for analyzing LIBS signals showcases an innovative method that excels in real-time operation and minimal data dependency. Their method's ability to operate in real-time and with reduced data dependency provides a compelling framework for future considerations in our work, especially when evaluating the balance between model complexity and operational efficiency in reducing overall error.
+
+\citeauthor{chen_xgboost_2016} introduced XGBoost, a scalable end-to-end tree boosting system that has gained widespread use among data scientists for its state-of-the-art performance in various machine learning challenges. The system is particularly notable for its novel sparsity-aware algorithm tailored for sparse data and the introduction of a weighted quantile sketch method for approximate tree learning. These advancements allow XGBoost to efficiently handle large-scale data with significantly reduced resource requirements. Utilizing cache access patterns, data compression, and sharding, the authors were able to significantly enhance the scalability and efficiency of XGBoost. These approaches enabled it to process a large number of examples with fewer resources compared to other exisiting systems\cite{chen_xgboost_2016}.
+
+While XGBoost's emphasis on scalability and computational efficiency diverges from our current work on enhancing the MOC model, the methodologies it employs could be instrumental for future developments in our research. The efficient handling of large datasets, a key feature of XGBoost, provides a framework that could be adapted to improve our model's performance with extensive data.
```

Comment:
Reads like ChatGPT's writing. 😬

Also, while I know why this was included, this XGBoost section really seems like the odd-one-out. Very strange. Feels like it comes out of nowhere. Should probably have included some details about the nature of the problem (multiple, multivariate regression) and the similarities between the methods (e.g. ensemble, also SM is conceptually kind of like boosting) and motivated it with that.